v63003/u_inref/gtm8735.csh

```diff
@@ -66,16 +66,23 @@ $DSE dump -file|&$grep "Read Only"
 
 echo '# Displaying status of gtmhelp databases to verify files have read-only permissions for user/group/other'
 $gtm_tst/com/lsminusl.csh $ydb_dist/*.dat|$tst_awk '{print $1,$9}'
-
+set i = 0
 foreach dir ($ydb_dist/*.gld)
 	echo "# Displaying status of "$dir
 	setenv ydb_gbldir $dir
 	$DSE dump -file |& $grep "Read Only"
 	echo "# Attempting to change to no read only"
-	$MUPIP SET -region DEFAULT -NOREAD_ONLY
+	set x = ""
+	while ("$x" == "")
+		@ i++
```

Comment:
We don't need an "i" variable for this. Use temp.out for all output instead of temp$i.out. Also if a READONLYLKFAIL error is seen, then redo the loop. Else break out of the loop. This way if we have any error other than READONLYLKFAIL or DBFILOPERR we still break out of the loop instead of incorrectly continuing the loop.